nexus/src/app/sagas/instance_create.rs

```diff
@@ -609,17 +607,40 @@ async fn sic_alloc_server(
     //   schedule instances that belong to a cluster on different failure
     //   domains. See https://github.com/oxidecomputer/omicron/issues/1705.
 
-    osagactx
+    // TODO: Fix these values. They're wrong now, but they let us move
+    // forward with plumbing.
+    let params = sagactx.saga_params::<Params>()?;
+    let hardware_threads = params.create_params.ncpus.0;
+    let rss_ram = params.create_params.memory;
+    let reservoir_ram = omicron_common::api::external::ByteCount::from(0);
```

Comment:
*right now*, our policy for "how many resources are consumed by instances" is an exact 1:1 mapping of:

- Guest CPUS -> Hardware threads
- Guest RAM -> RSS RAM

Arguably, propolis will consume some overhead, and we will need to make considerations for the reservoir cost, in addition to the "rss ram" cost.

However, I wanted to put this PR up as-is, since it'll mechanically be very similar - just a difference of accounting.